libs/wire-api/src/Wire/API/SwaggerHelper.hs

```diff
@@ -47,3 +57,49 @@ cleanupSwagger =
       (S.properties . traverse . S._Inline %~ sanitise)
         . (S.required %~ nubOrd)
         . (S.enum_ . _Just %~ nub)
+    -- servant-openapi and servant-swagger both insert default responses with codes 404 and 400.
+    -- They have a simple structure that we can match against, and remove from the final structure.
+    stripDefaultErrors :: HttpStatusCode -> Referenced Response -> Responses' -> Responses'
```

Comment:
Since we're already patching the library, would it make sense to also patch in settings to disable this default responses?

Reply:
i'd prefer to do that sepately.  this change is big enough, and we have had odd defaults until now, too.